Adapter/ShopwareAdapter/RequestGenerator/Product/ConfiguratorSet/ConfiguratorSetRequestGenerator.php

```diff
@@ -10,6 +10,10 @@
  */
 class ConfiguratorSetRequestGenerator implements ConfiguratorSetRequestGeneratorInterface
 {
+    const STANDARD = 0;
```

Comment:
Bitte in das TransferObject verschieben, Validierung einbauen und jeweils in den Adaptern jeweils auf diesen Wert mappen. Anstelle des integers bitte einen string verwenden (dropdown, image) und den standard Wert bitte entsorgen. In Shopware ist das ja dropdown :)

Reply:
In Shopware gibt es aber 3 ;)

Standard ist zwar ein dropdown, verhält sich aber etwas anders.

https://community.shopware.com/Artikel-Varianten_detail_920.html#Art_der_Varianten-Darstellung

Reply:
Dann nenn es bitte einfach nicht standard :)